.github/ISSUE_TEMPLATE/publish-a-component.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "Publish a component"
+about: Everything we need to do when we publish a component
+title: ''
+labels: "contribution or major iteration, epic"
+assignees: ''
+
+---
+
+## What
+<!-- Add name of component, plus any relevant links (for example, community backlog issues and comments, pull requests for the component and its guidance) -->
+
+## Why
+<!-- Add reason for publishing the component -->
+
+## Who needs to know about this
+<!-- Add team roles involved in publishing the component -->
+
+## Acceptance criteria
+<!-- Customise, and add component-specific checklist items to, this list of criteria all components need to meet -->
+We need to:
+- [ ] document any decisions we've made
```

Comment:
Where? [Decision records](https://github.com/alphagov/govuk-design-system-architecture/tree/main/decision-records)? Or in the 'Published' column on the [GitHub backlog](https://github.com/alphagov/govuk-design-system-backlog/projects/1)?